### DISCUSSION TITLE
Goal 9: Build resilient infrastructure, promote inclusive and sustainable industrialization, and foster innovation

### DISCUSSION THREAD

**Discussion by HSCunha** [Project-Resilience/platform#31] [General]
Goal 9: [Build resilient infrastructure, promote inclusive and sustainable industrialization, and foster innovation](https://github.com/Project-Resilience/platform/blob/main/goals/goal_09.md)

**Comment by ofrancon** [comment]
This work could be useful for hurricane preparedness: https://deepmind.google/discover/blog/weather-lab-cyclone-predictions-with-ai/
Could be used to:
- organize evacuation efforts
- position repair crews closer to the cyclone path, e.g. for energy, telecom, water networks
- prepare water, food and first aid distribution

**Comment by GPaolo** [comment]
This is the link for the TELCO challenge for using LLM for network issue troubleshooting: 
- https://aiforgood.itu.int/event/ai-telco-troubleshooting-challenge-global-launch/
- https://www.mwcdoha.com/articles/the-ai-telco-troubleshooting-challenge-launches-to-transform-network-reliability

The goal of the challenge is to finetune off-the-shelf LLM for helping technicians to troubleshoot issues in telco networks.
They have a dataset TeleLogs: https://huggingface.co/datasets/netop/TeleLogs with 4000 datapoints
There is also a leaderboard: https://huggingface.co/spaces/otellm/leaderboard where more datasets are listed as well

There are 3 categories:
- **Generalisation to New Faults** will look at LLMs for RCA; 
- **Small Models at the Edge** is about lightweight edge-deployable models; 
- **Explainability and Reasoning** will focus on the AI systems that explain their reasoning. 
The judging criteria will evaluate entrants for the accuracy, efficiency, reasoning capability and security considerations of their models.

This is organized for the AI For Good summit and is a challenge that is live between the 28 Nov 2025 till the 02 Feb 2026

**1st place in each track will win.**

€35 000 Euros to be equally shared between the winners for each of the competition tracks, which includes cash prizes, travel, and accommodation to attend [MCW Barcelona](https://www.mwcbarcelona.com/) in March 2026 and present the work to the AI for network community. Tickets will be also offered to the winners.
